readme.md

```diff
@@ -47,10 +47,10 @@ Gradle:
       jcenter()
     } 
     dependencies {
-      compile "com.palantir.remoting:http-clients:0.1.2"
-      compile "com.palantir.remoting:http-servers:0.1.2"
-      compile "com.palantir.remoting:trust-stores:0.1.2"
-      compile "com.palantir.remoting:error-handling:0.1.2"
+      compile "com.palantir.remoting:http-clients:$version"
+      compile "com.palantir.remoting:http-servers:$version"
+      compile "com.palantir.remoting:ssl-config:$version"
+      compile "com.palantir.remoting:error-handling:$version"
     }
```

Comment:
The README currently doesn't have any actual usage examples, so may punt on this (unit tests should provide a fairly good example of how to do this)